stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -14,6 +14,13 @@ class AsyncResult():
     def ready(self) -> bool: ...
     def successful(self) -> bool: ...
 
+class Event():
+    def __init__(self, *, ctx: Any) -> None: ...
```

Comment:
I think you are right.  I was originally thinking about being more specific for ctx also, but was trying to minimize the change.  I noticed __init__ in Pool was using this format for context so copied that.  I think it would be best if I add the BaseContext stub.  I'll update shortly.